hbase-server/src/main/java/org/apache/hadoop/hbase/master/ServerManager.java

```diff
@@ -649,10 +649,9 @@ public synchronized void moveFromOnlineToDeadServers(final ServerName sn) {
   public synchronized boolean removeServerFromDrainList(final ServerName sn) {
     // Warn if the server (sn) is not online. ServerName is of the form:
     // <hostname> , <port> , <startcode>
-
     if (!this.isServerOnline(sn)) {
-      LOG.warn("Server " + sn + " is not currently online. "
-        + "Removing from draining list anyway, as requested.");
+      LOG.warn(
```

Comment:
Please don't clean up all these logger usages in files that you don't otherwise touch. Lets fix them up in a dedicated PR.